Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml

```diff
@@ -0,0 +1,652 @@
+  description: The URL.
+- contextPath: VMRay.Sample.IOC.URL.Operation
+  description: The operation of the specified URL.
+- contextPath: VMRay.Sample.IOC.URL.ID
+  description: The ID of the URL.
+- contextPath: VMRay.Sample.IOC.URL.Type
+  description: The type of the URL.
+- contextPath: VMRay.Sample.IOC.Domain.AnalysisID
+  description: The IDs of the other analyses that contain the given domain.
+- contextPath: VMRay.Sample.IOC.Domain.Domain
+  description: The domain.
+- contextPath: VMRay.Sample.IOC.Domain.ID
+  description: The ID of the domain.
+- contextPath: VMRay.Sample.IOC.Domain.Type
+  description: The type of the domain.
+- contextPath: VMRay.Sample.IOC.IP.AnalysisID
+  description: The IDs of the other analyses that contain the given IP address.
+- contextPath: VMRay.Sample.IOC.IP.IP
+  description: The IP address.
+- contextPath: VMRay.Sample.IOC.IP.Operation
+  description: The operation of the given IP address.
+- contextPath: VMRay.Sample.IOC.IP.ID
+  description: The ID of the IP address.
+- contextPath: VMRay.Sample.IOC.IP.Type
+  description: The type of the IP address.
+- contextPath: VMRay.Sample.IOC.Mutex.AnalysisID
+  description: The IDs of other analyses that contain the given IP address.
+- contextPath: VMRay.Sample.IOC.Mutex.Name
+  description: The name of the mutex.
+- contextPath: VMRay.Sample.IOC.Mutex.Operation
+  description: The operation of the given mutex
+- contextPath: VMRay.Sample.IOC.Mutex.ID
+  description: The ID of the mutex.
+- contextPath: VMRay.Sample.IOC.Mutex.Type
+  description: The type of the mutex.
+- contextPath: VMRay.Sample.IOC.File.AnalysisID
+  description: The IDs of other analyses that contain the given file.
+- contextPath: VMRay.Sample.IOC.File.Name
+  description: The name of the file.
+- contextPath: VMRay.Sample.IOC.File.Operation
+  description: The operation of the given file.
+- contextPath: VMRay.Sample.IOC.File.ID
+  description: The ID of the file.
+- contextPath: VMRay.Sample.IOC.File.Type
+  description: The type of the file.
+- contextPath: VMRay.Sample.IOC.File.Hashes.MD5
+  description: The MD5 hash of the given file.
+- contextPath: VMRay.Sample.IOC.File.Hashes.SSDeep
+  description: The SSDeep hash of the given file.
+- contextPath: VMRay.Sample.IOC.File.Hashes.SHA256
+  description: The SHA256 hash of the given file.
+- contextPath: VMRay.Sample.IOC.File.Hashes.SHA1
+  description: The SHA1 hash of the given file.
+- contextPath: VMRay.ThreatIndicator.AnalysisID
+  description: The list of connected analysis IDs.
+- contextPath: VMRay.ThreatIndicator.Category
+  description: The category of threat indicators.
+- contextPath: VMRay.ThreatIndicator.Classification
+  description: The classifications of threat indicators.
+- contextPath: VMRay.ThreatIndicator.ID
+  description: The ID of the threat indicator.
+- contextPath: VMRay.ThreatIndicator.Operation
+  description: The operation that caused the indicators.
+fromversion: 5.5.0
+tests:
+- no test
```

Comment:
I see that there is VMRay-Test-URL playbook, should he test the VMRay-Detonate-URL playbook? if so please add him.

Reply:
Done in1faa07e1a7c353b5645af0f521e78c209fe3ac92